src/main/java/uk/gov/hmcts/reform/logging/exception/AbstractLoggingException.java

```diff
@@ -47,9 +46,7 @@ private static void triggerBadImplementationLog(Throwable cause) {
         log.error("Bad implementation of '" + cause.getClass().getCanonicalName() + "' in use", invalid);
     }
 
-    public static AbstractLoggingException getFromLogEvent(ILoggingEvent event) {
-        ThrowableProxy proxy = (ThrowableProxy) event.getThrowableProxy();
-
+    public static AbstractLoggingException getFromLogEvent(ThrowableProxy proxy) {
```

Comment:
I think we should change the method name as well, as it no longer operates on a log event.

Reply:
oopsie